src/Fabulous.AST/Widgets/TypeDefinitions/Abbrev.fs

```diff
@@ -38,27 +38,27 @@ module Alias =
             ))
 
 [<AutoOpen>]
-module AliasBuilders =
+module AbbrevBuilders =
```

Comment:
Nitpick, but always try to do one thing in a PR.
This will lead to a nicer Git history.
In a couple of years, people will be confused why this was changed as part of an `If Expression support` PR.

Reply:
Too late :). I will consider it for the future